### PR TITLE
O9S: fix stack marker regex and entropy bug

### PR DESCRIPTION
**Pull request by DrLippe**
this should resolve quisquous/cactbot#334

### CODE REVIEW

File: ui/raidboss/data/triggers/o9s.js [hunk 1]
```diff
@@ -333,7 +337,7 @@
     },
     {
       id: 'O9S Knock Down Marker',
-      regex: / 1B:........:(\y{Name}):0000:0000:0057:0000:0000:0000:/,
+      regex: / 1B:........:(\y{Name}):0000:E000:0057:0000:0000:0000:/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o9s.js]: I think these should really be `/ 1B:........:(\y{Name}):....:....:0057:/,`, and the equivalent for the `003E` marker.

The first two things after the name aren't consistent (and so I think it was a mistake to specify them here), and although the last ones are always zeroes, maybe better to underspecify.

File: ui/raidboss/data/triggers/o9s.js [hunk 2]
```diff
@@ -172,7 +172,7 @@
     // Fire Path
     {
       id: 'O9S Fire Phase Tracking',
-      regex: / 14:3186:Chaos starts using Blaze /,
+      regex: / 14:3186:Chaos starts using Blaze/,
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o9s.js]: This isn't a fix, just a cleanup, right? I think the space is still correct even if messy?

File: ui/raidboss/data/triggers/o9s.js [hunk 3]
```diff
@@ -221,6 +221,10 @@
           fr: 'Packez-vous et restez',
         };
       },
+      run: function(data) {
+        if (data.phaseType == 'orb' || data.entropyCount == 2)
+          delete data.entropyCount;
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o9s.js]: Aha, whoops! Thanks for this fix.

File: ui/raidboss/data/triggers/o9s.js (hunk 1 above)
Comment (DrLippe) [reply ui/raidboss/data/triggers/o9s.js]: Honestly, no idea what these stand for but they changed on recent FFXIV plugin update.

File: ui/raidboss/data/triggers/o9s.js (hunk 2 above)
Comment (DrLippe) [reply ui/raidboss/data/triggers/o9s.js]: The log line says ` 14:3186:Chaos starts using Blaze on Chaos.` so the white space is fine for now.
Maybe the ` on Chaos.` part will disappear some day?

**Comment by quisquous** [comment]
Thanks!

**Pull request merged**